CITATION.cff

```diff
@@ -0,0 +1,38 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: 'NetCDF-C '
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - name: NSF Unidata
```

Comment:
I would like it if we could list the authors, in addition to saying NFS Unidata.

Reply:
That is the 'waiting to hear back' part referenced above, we will be listing the authors in the final version :).